docs/v1/Makefile

```diff
@@ -23,5 +22,11 @@ ${SPEC}.html: ${SPEC}.adoc
 	-r asciidoctor-lists \
 	-a rouge-css=$(ROUGE_CSS) $<
 
+images:
+	soffice --convert-to png --outdir resources/figs resources/figs/*.odg > /dev/null 2>&1
```

Comment:
svg are also generated by the current docs/v1/Makefile.  Are they no longer necessary with AsciiDoc PDF and HTML?